src/popup/utils/utils.ts

```diff
@@ -374,7 +374,7 @@ export function categorizeContractCallTxObject(transaction: ITransaction): {
   url?: string
   note?: string
 } | null {
-  if (!compareCaseInsensitive(transaction.tx.type, SCHEMA.TX_TYPE.contractCall)) {
+  if (!compareCaseInsensitive(transaction.tx.type, Tag[Tag.ContractCallTx])) {
```

Comment:
Maybe the `tx.type` can be now compared directly to the `Tag[Tag.ContractCallTx]`. I mean do they have the same casing?

Reply:
As discussed, I have added code comments for the usage of tx `type`. We need to support case sensitive contract call URLs